conf/configure.fv3.orion.intel

```diff
@@ -39,7 +39,7 @@ include       $(ESMFMKFILE)
 ESMF_INC    = $(ESMF_F90COMPILEPATHS)
 
 NEMSIOINC = -I$(NEMSIO_INC)
-NCEPLIBS = $(POST_LIB) $(NEMSIO_LIB) $(G2_LIB4) $(G2TMPL_LIB) $(BACIO_LIB4) $(SP_LIBd) $(W3EMC_LIBd) $(W3NCO_LIBd) $(CRTM_LIB) $(JASPER_LIB) -lpng -lz
```

Comment:
So is JASPER_LIB defined in hpc-stack? what is the value of it?

Reply:
JASPER_LIB is not defined in hpc-stack.  However, JASPER_ROOT is defined.